### ISSUE TITLE
use default aliases from vite when defined

### ISSUE THREAD

**Issue by lifeiscontent** [eryue0220/unplugin-stylex#54]
it seems like if I've already defined my `alises` in my vite config, they don't get automatically picked up by this plugin, would be nice.

**Comment by eryue0220** [comment]
Could you please provide more detail about this problem?

**Comment by lifeiscontent** [comment]
@eryue0220 yep, given this remix template:

https://github.com/remix-run/remix/blob/main/templates/remix

in the https://github.com/remix-run/remix/blob/main/templates/remix/vite.config.ts

there's a call to `tsconfigPaths()`

which will automatically load these paths into vite's aliases:

https://github.com/remix-run/remix/blob/main/templates/remix/tsconfig.json#L26

if this plugin had better vite support, it would take advantage of that configuration rather than having to redefine it.

**Comment by eryue0220** [comment]
@lifeiscontent  thank you for your feedback, I'll give you a feedback soon.

**Comment by eryue0220** [comment]
Hi, @lifeiscontent  I've checked the above example, and I think you could still import `vite-tsconfig-paths` in your project's `vite.config.js`. `unplugin-stylex` is just a plugin for `stylex`.

**Comment by lifeiscontent** [comment]
yeah, and their is an option for stylex called `alieses` which should probably map to the `alieses` previously defined outside of the plugin which is what my ask is. https://vitejs.dev/config/shared-options#resolve-alias

**Comment by eryue0220** [comment]
I got it. Thank you for your feedback, would you like to contribute this feature?
